examples/example_landing_zone_reader/README.md

```diff
@@ -25,18 +24,19 @@ resource "aws_lambda_function" "hello_world" {
   function_name = "my_hello_world"
   description   = "Managed by TerraHub"
   runtime       = "nodejs10.x"
-  handler       = "handler.index"
+  handler       = "index.handler"
   memory_size   = "128"
   timeout       = "30"
   role          = "arn:aws:iam::123456789012:role/ServiceRoleForLambdaEdge"
   s3_bucket     = "www.terrahub.io"
   s3_key        = "/hello-world/nodejs10.x.zip"
 
   vpc_config = {
-    subnet_ids         = ["${local.landing_zone_subnet_ids["default"]}"]
-    security_group_ids = ["${local.landing_zone_security_group_ids["default"]}"]
+    subnet_ids         = [local.landing_zone_subnet_ids["default"]]
+    security_group_ids = [local.landing_zone_security_group_ids["default"]]
```

Comment:
Please change this line with `security_group_ids = local.landing_zone_security_group_ids["default"]`